api_test.rb

```diff
@@ -1,40 +1,57 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+require "benchmark"
```

Comment:
Add benchmarking to the API test. Seems a useful thing to see every time we run the API test?